src/main/java/jose/task/Deadline.java

```diff
@@ -0,0 +1,51 @@
+package jose.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDateTime by;
+
+    /**
+     * Constructor that sets isDone to false.
+     *
+     * @param description Task description.
+     */
+    public Deadline(String description, String by) {
+        super(description, false);
+        String[] dateTime = by.split(" ");
+        this.by = LocalDateTime.of(LocalDate.parse(dateTime[0]),
+                LocalTime.parse(dateTime[1], DateTimeFormatter.ofPattern("HHmm")));
+    }
+
+    /**
+     * Constructor that sets all variables.
+     *
+     * @param description Task description
```

Comment:
Do you think a punctuation is missing here?

I notice similar issues in the other classes.